io/src/FileSystem/private/FileSystemManager_p.cpp

```diff
@@ -79,8 +79,6 @@ bool FileSystemManager::Private::deregisterFileSystem(const std::string & author
 	}
 
 	if(found == false) {
```

Comment:
This can just be replaced by `return found;`

Reply:
updated.